app/src/main/kotlin/co/netguru/baby/monitor/client/feature/server/ChildMonitorFragment.kt

```diff
@@ -254,7 +253,7 @@ class ChildMonitorFragment : BaseDaggerFragment(), ServiceConnection {
 
     private fun handleWebSocketAction(ws: WebSocket, key: String, value: String) {
         when (key) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/59ecd15255bdd529a391ea0d5072dc37d88e9138/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/server/ChildMonitorFragment.kt#L255"></span>Prefer using ‘if’ for binary conditions instead of ‘when’.</td>
    </tr>
  </tbody>
</table>